src/framebuffer.rs

```diff
@@ -0,0 +1,123 @@
+use core::ptr;
+use core::slice;
+use header::Tag;
+
+#[derive(Debug, PartialEq)]
+pub struct FramebufferTag<'a> {
```

Comment:
Shouldn't this be `repr(C, packed)`?

Reply:
As far as I can see, this doesn't follow the representation in memory and is created using `Reader`, so it doesn't need to be

Reply:
Yeah you're right.  `Reader` is nice in that regard.

The only place we need `repr(C, packed)` is `FrameBufferColor` since we grab a slice of that memory, and it's already there.

Sorry for the noise!